media-plugins/vdr-ddci2/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>d.scheller.oss@gmail.com</email>
+		<name>Daniel Scheller</name>
+	</maintainer>
+	<upstream>
```

Comment:
For the future: please remember to add proxy-maint, so we know to commit for you ;-). I'll fix it now.